sharepoint/sharepoint-ps/sharepoint-online/Export-SPODataAccessGovernanceInsight.md

```diff
@@ -0,0 +1,73 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
+schema: 2.0.0
+author: pvrk
+ms.author: pvrk
+manager: 
+ms.reviewer:
+---
+
+# Export-SPODataAccessGovernanceInsight
+
+## SYNOPSIS
+
+This cmdlet downloads the Data Access Governance (DAG) reports to the current working directory
+
+## SYNTAX
+
+```
+Export-SPODataAccessGovernanceInsight -ReportID <Guid> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This cmdlet exports or downloads the DAG report, specified by the ReportID, to the current working directory. The ReportID is shown in the output of the 'Start-SPODataAccessGovernanceInsight' command. It can also be fetched from the output of the 'Get-SPODataAccessGovernanceInsight' command.
```

Comment:
>'Get-SPODataAccessGovernanceInsight' [](http://example.com/codeflow?start=247&length=36)

2 options here:
1. use backticks as well for this cmdlet name
2. link this cmdlet to your documentation, i.e.

```
[Get-SPODataAccessGovernanceInsight](./Get-SPODataAccessGovernanceInsight.md)
```